packages/nimble-components/src/banner/styles.ts

```diff
@@ -183,6 +196,10 @@ export const styles = css`
                 Theme.color,
                 css`
                     :host {
```

Comment:
Do you have to duplicate the same `color` theme styling for each severity if the styles don't change. I'm not familiar with the expected patterns of using `MultivaluePropertyStyleSheetBehavior`, but I would have thought you could leave off the `color` theme behavior and it would just use the styling defined outside of the `MultivaluePropertyStyleSheetBehavior`. Is that not the case?

Reply:
Duh. Yes, I can avoid that duplication for the `color` theme.